doc/widgets.md

```diff
@@ -1,22 +1,31 @@
 # Widgets
 
-One cool feature when developing games with Flutter is the ability to use Flutter's extensive toolset for building UIs, Flame tries to expand on that by introducing widgets that are made with games in mind.
+One cool feature when developing games with Flutter is the ability to use Flutter's extensive
```

Comment:
Again with the cool/useful feature hehe
How about:
One advantage when...

Reply:
I agree (not written by me), will change those.